src/relayclient/AccountManager.ts

```diff
@@ -7,25 +7,36 @@ import { getEip712Signature, isSameAddress } from '../common/utils'
 import { Address } from './types/Aliases'
 import { PrefixedHexString } from 'ethereumjs-tx'
 import { AccountManagerConfig } from './GSNConfigurator'
+import { HttpProvider } from 'web3-core'
+import Web3 from 'web3'
 
 export interface AccountKeypair {
   privateKey: Buffer
   address: Address
 }
 
+function toAddress (wallet: any): string {
+  // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+  return `0x${wallet.getAddress().toString('hex')}`
```

Comment:
instead of disable typescript, simply use `'0x'+xxx.toString()`

Reply:
That is not a solution, it is hiding the problem. The problem is that wallet does not have types. It is not that hard to declare the types for it, which is a solution that I think we should do.

Reply:
good idea. regardless, string concat is clearer